src/config/env.json

```diff
@@ -6,7 +6,9 @@
     "DD_TRACE_AGENT_PORT": 8126,
     "BIO_AUTH_SERVICE_BACKEND": "template",
     "BIO_AUTH_SERVICE_URL": "http://protocol-bio-auth-service:3005",
+    "INTEGRATION_CONTROLLER": "http://rnp-controller/3016",
```

Comment:
This should be removed from this repo and set as an environment variable in the appropriate environment as it is not a generic value.

Reply:
done